src/load/load/load.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import hvac
+
+import load.context
+
+from lib.util import (
+  url_host_port
+)
+from lib.vault import Vault
+
+
+def load_secrets(artsy_project, vault_host, vault_port, secrets_file, kvv2_mount_point):
+  ''' load secrets from  Vault and write them to a file '''
+  logging.debug(f'Loading secrets from {vault_host}:{vault_port} under {kvv2_mount_point}')
+
+  path = f'kubernetes/apps/{artsy_project}/'
+
+  vault_client = Vault(
+    url_host_port(vault_host, vault_port),
+    auth_method='kubernetes',
+#    auth_method='iam',
```

Comment:
This looks like dev code, or do you use it for testing?

Reply:
exactly. for local testing i use iam. okay. let me remove iam for now.

Reply:
Either that or you can use an env flag to switch to `iam` in development mode

Reply:
oh yes. let me try `if env == 'dev' ...`

Reply:
i'd have to introduce a separate arg or env var. cause i need it to mean "the env i am running the script in", whereas the current `env` arg means "the env to operate migration in".

Reply:
opted to add an `auth` [arg](https://github.com/artsy/fortress/pull/1/commits/8000caa370685c6778a283b1b7e550f800c5b589), since even in local dev one may auth using iam or token.